docker-compose.yaml

```diff
@@ -4,32 +4,15 @@ services:
   point_node:
     image: pointnetwork/pointnetwork_node:$POINT_NODE_VERSION
     container_name: point_node
-    entrypoint: 'npm'
-    command: 'run start:docker'
     volumes:
       - point_contracts:/app/truffle/build/contracts
       - point_node_data:/data
       - $POINT_KEYSTORE:/data/keystore
-      - $POINT_DEPLOYSPACE:/app/deployspace:ro
```

Comment:
Why did you decide to remove this?

Reply:
> It's never used anywhere.

From what I can see this is not true. Please take a look at the `docker-compose.yaml` and `deploy-from-deployspace.sh` files.